crypto/ocsp/ocsp_vfy.c

```diff
@@ -203,10 +218,10 @@ static X509 *ocsp_find_signer_sk(STACK_OF(X509) *certs, OCSP_RESPID *id)
 
 static int ocsp_check_issuer(OCSP_BASICRESP *bs, STACK_OF(X509) *chain)
 {
-    STACK_OF(OCSP_SINGLERESP) *sresp;
+    STACK_OF(OCSP_SINGLERESP) *sresp = bs->tbsResponseData.responses;
     X509 *signer, *sca;
     OCSP_CERTID *caid = NULL;
-    int i;
+    int ret;
 
     sresp = bs->tbsResponseData.responses;
```

Comment:
This line can go now right?

Reply:
Right, so removed the duplicate assignment.